usecases/list_available_nodes.py

```diff
@@ -0,0 +1,12 @@
+from typing import List, Union
+
+from gateways.cache.cache_gateway import CacheGateway
+
+
+class ListAvailableNodes:
```

Comment:
This class seems to be using a different naming convention than most of the code.
If I understand correctly, the concept of `Node` here is the same thing that is being called `Network` in other parts of the code.

Is that right?

If so, I would suggest adopting the name `Node` everywhere.

Reply:
I've changed everything to `Node`